corehq/apps/userreports/expressions/specs.py

```diff
@@ -239,3 +240,27 @@ def __call__(self, item, context=None):
         for property_name, expression in self._compiled_properties.items():
             ret[property_name] = expression(item, context)
         return ret
+
+
+class EvalExpressionSpec(JsonObject):
+    type = TypeProperty('evaluator')
+    statement = StringProperty(required=True)
+    context_variables = DictProperty(required=True)
+
+    def configure(self, context_variables):
+        self._context_variables = context_variables
```

Comment:
(not actionable) this whole pattern of the `_property` being the compiled expression and the manual steps needed to get there bugs me. we should figure out a way to clean up UCR to make that better but it's definitely out of scope for this PR.